client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx

```diff
@@ -103,6 +104,19 @@ const PlansWrapper: React.FC< Props > = ( props ) => {
 			} );
 		}
 
+		const cartItemForStorageAddOn = cartItems?.find(
+			( items ) => items.product_slug === PRODUCT_1GB_SPACE
+		);
+
+		if ( cartItemForStorageAddOn ) {
+			setProductCartItems( [ cartItemForStorageAddOn ] );
+
+			cartItemForStorageAddOn?.extra &&
+				recordTracksEvent( 'calypso_signup_storage_add_on_upgrade_click', {
```

Comment:
Thanks for the suggestion 

I actually took a closer look and verified this in testing, but the tracks event logic I included here is redundant https://github.com/Automattic/wp-calypso/pull/83771#discussion_r1388460589. Because of this, I remove it entirely

It should be handled by existing logic in https://github.com/Automattic/wp-calypso/blob/b9d11f5e83540a22ecdbf6a237839d684d4c9977/client/my-sites/plans-features-main/index.tsx#L342-L346